src/base.js

```diff
@@ -13,8 +13,225 @@
  * limitations under the License.
  */
 
+var Util = require('./util.js');
+
+// Start at a higher number to reduce chance of conflict
+var nextDisplayId = 1000;
+
+/**
+ * The base class for all VR displays.
+ */
+function VRDisplay() {
+  this.displayId = nextDisplayId++;
+  this.displayName = 'webvr-polyfill displayName';
+
+  this.isConnected = true;
+  this.isPresenting = false;
+  this.capabilities = {
+    hasPosition: false,
+    hasOrientation: false,
+    hasExternalDisplay: false,
+    canPresent: false
+  };
+  this.stageParameters = null;
+
+  // "Private" members.
+  this.waitingForPresent_ = false;
+  this.layer_ = null;
+
+  this.fullscreenEventTarget_ = null;
+  this.fullscreenChangeHandler_ = null;
+  this.fullscreenErrorHandler_ = null;
+}
+
+VRDisplay.prototype.requestAnimationFrame = function(callback) {
+  return window.requestAnimationFrame(callback);
+};
+
+VRDisplay.prototype.cancelAnimationFrame = function(id) {
+  return window.cancelAnimationFrame(id);
+};
+
+VRDisplay.prototype.requestPresent = function(layer) {
+  var self = this;
+  this.layer_ = layer;
```

Comment:
ok, that's fair. could we add [a comment here](https://github.com/borismus/webvr-polyfill/pull/33/files#diff-d7e697eff3913f1acaacac8002c3b05eR158) about using an array?